src/test/scala/DictionaryTest.scala

```diff
@@ -0,0 +1,18 @@
+import org.scalatest.WordSpecLike
+import Dictionary._
+
+class DictionaryTest extends WordSpecLike {
+
+  "DictionaryTest" should {
+
+    "extractUniqueWordsFromTextFile" should {
+
+      "return 6 unique words" in {
+        assertResult(6){
+          extractUniqueWordsFromTextFile("myWords").size
```

Comment:
Good Test! Maybe can we assert on the 6 words that are returned?
In other words, it should return "banana", "dfdbkjdflg", etc, etc